wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/document/DocumentInfoScreen.kt

```diff
@@ -51,11 +51,21 @@ import com.android.identity_credential.wallet.ui.KeyValuePairText
 import com.android.identity_credential.wallet.ui.ScreenWithAppBarAndBackButton
 import com.android.identity_credential.wallet.ui.durationFromNowText
 import kotlinx.coroutines.launch
-import java.util.Locale
+import java.time.Instant
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.time.format.DateTimeFormatter
+import java.util.*
 
 
 private const val TAG = "DocumentInfoScreen"
 
+fun formatDate(timestamp: Long): String {
+    val dateTime = LocalDateTime.ofInstant(Instant.ofEpochSecond(timestamp), ZoneId.systemDefault())
+    val formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss", Locale.getDefault())
+    return dateTime.format(formatter)
+}
```

Comment:
Please don't use Java routines for this, it'll just create work when we port this to Kotlin Multiplatform.  Also, if you create a helper it needs to be `private` (otherwise it pollutes the package) and should also take a `kotlinx.datetime.Instant`, not a `Long`.

In this case, I would just use the existing `Instant.asFormattedDateTimeInCurrentTimezone` extension defined in [CommonExtensions.kt](https://github.com/openwallet-foundation-labs/identity-credential/blob/main/wallet/src/main/java/com/android/identity_credential/wallet/util/CommonExtensions.kt#L44).

Reply:
Done.